go.mod

```diff
@@ -1,27 +1,30 @@
 module github.com/corazawaf/coraza-spoa
 
-go 1.19
+go 1.23.1
```

Comment:
With go 1.23.1 the application fails to start for me.
With 1.22.3 (as used by the waf-core https://github.com/corazawaf/coraza/blob/main/go.mod#L3) it works

Error:
`Oct 06 16:43:57 lb01 coraza[79533]: {"level":"fatal","error":"initializing application '\\x00': invalid WAF config from string: failed to readfile: read /etc/coraza-spoa/coraza.conf: invalid argument","time":"2024-10-06T16:43:57+02:00","message":"Failed creating applications"}`

Reply:
same error with go 1.23.2 :
9:45PM FTL Failed creating applications error="initializing application '\\x00': invalid WAF config from string: failed to readfile: read /opt/coraza-spoa/etc/coraza.conf: invalid argument"

Reply:
Intersting. Will try to reproduce